### PR TITLE
Added support for directories #1

### PR DESCRIPTION
**Pull request by nclsbayona**
I would suggest changing the sys.argv approach to argparser or argh or something like those

### CODE REVIEW

File: JSONCleaner.py [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+from os import listdir
+
+# Determine if it's or not a directory
+def isDirectory(path):
+    if (isdir(path)):
+        return True
+    return False
 
 # Make the JSON file pretty
 def FormatJSONFile(PathToJSONFile):
```
Comment (Tresquel) [inline JSONCleaner.py:13]: We could merge this function into the `formatJson` one

File: JSONCleaner.py [hunk 2]
```diff
@@ -8,20 +16,57 @@ def FormatJSONFile(PathToJSONFile):
...
+            file_path=join(path, file)
+            if (isDirectory(file_path)):
+                deformatJson(file_path)
+            else:
+                DeFormatJSONFile(file_path)
+    else:
+        DeFormatJSONFile(path)
 
 # De-Pretty the JSON file
 def DeFormatJSONFile(PathToJSONFile):
```
Comment (Tresquel) [inline JSONCleaner.py:52]: We could merge this function into 'deformatJson` too

**Review by Tresquel — CHANGES_REQUESTED** [review]
First up thank you so much for contributing but there's a few things we could change.

**Comment by Tresquel** [comment]
And as you said we could try something like `argparse` or `argh`.

**Comment by nclsbayona** [comment]
I'm sorry, I didn't mean it was ready, I misclicked

**Comment by Tresquel** [comment]
You created a second pull request which was merged so I'm gonna close this one

**Pull request closed without merging**